releases/release-1.31/release-notes/maps/pr-125263-map.yaml

```
@@ -0,0 +1,5 @@
+pr: 125263
+releasenote:
+  text: Fix bug where Server Side Apply causes spurious resourceVersion bumps on no-op
```

I don't think this was resolved?
```suggestion
  text: Fixed bug where Server Side Apply causes spurious resourceVersion bumps on no-op
```